book/_config.yml

```
@@ -38,6 +38,8 @@ execute:
     - "**/dataintegration-1.ipynb"
     - "**/dataintegration-2.ipynb"
     - "**/geospatial-advanced.ipynb"
+    - "**/Visualization_Earth_Engine_geemap_IS2_HW_2022.ipynb"
+    - "**Visualization_geemap_use_examples.ipynb"
```

Missing slash causing the builds to break.
```suggestion
    - "**/Visualization_geemap_use_examples.ipynb"
```